.buildkite/test-pipeline.yaml

```diff
@@ -796,7 +852,8 @@ steps:
   - pytest -v -s distributed/test_pp_cudagraph.py
   - pytest -v -s distributed/test_pipeline_parallel.py
 
-- label: LoRA TP Test (Distributed)
+- label: LoRA TP Test (Distributed) # 17 min
+  timeout_in_minutes: 50
```

Comment:
but this one looks kind of big?

Reply:
I fixed it, think it was a mis-copy from above.